app.js

```diff
@@ -88,8 +105,12 @@ function dispayCurrentGameEl() {
 
 function displayAllGames() {
     // clear out the past games list in the DOM
-
+    pastGamesEl.textContent = '';
     // loop through the past games in state
+    for (let game of pastGames) {
+        const gameEl = renderGame(game.name1, game.name2, game.score1, game, score2);
```

Comment:
The renderGame() function only takes 4 arguments, but you're giving it 5 here...